From 8a5b946f1c4a14c9a23ea8819cd43d37c9682d45 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Wed, 15 Jan 2025 00:52:24 -0700
Subject: [PATCH] debug-explicitly-init-variables

---
 src/ff/ffinit.f | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ff/ffinit.f b/src/ff/ffinit.f
index 7602e3c..6ba3ad5 100644
--- a/src/ff/ffinit.f
+++ b/src/ff/ffinit.f
@@ -776,6 +776,7 @@
 *
 *	first try - my home directory
 	path = '/user/gj/lib/'
+	fullname = ' '
 	fullname = path(1:index(path,' ')-1)//name
 	open(ifile,file=fullname,status='OLD',err=30)
 	return
-- 
2.47.1

